dashboard/src/actions/apps.ts

```diff
@@ -50,12 +55,9 @@ export const requestRollbackApp = createAction("REQUEST_ROLLBACK_APP");
 
 export const receiveRollbackApp = createAction("RECEIVE_ROLLBACK_APP_CONFIRMATION");
 
-export const errorApps = createAction("ERROR_APPS", resolve => {
-  return (err: Error) => resolve(err);
-});
-
-export const errorDeleteApp = createAction("ERROR_DELETE_APP", resolve => {
-  return (err: Error) => resolve(err);
+export const errorApp = createAction("ERROR_APP", resolve => {
+  return (err: FetchError | CreateError | UpgradeError | RollbackError | DeleteError) =>
```

Comment:
Similarly, we could just do `(err: CustomError)` here, or if we want to not include the existing servererror, define all these app errors as extending an `AppError` and use that? (At least, I assume the typescript annotation here takes that into account, haven't tried it).